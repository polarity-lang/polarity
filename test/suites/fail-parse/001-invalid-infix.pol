infix x + C(y) := D(x, y)
