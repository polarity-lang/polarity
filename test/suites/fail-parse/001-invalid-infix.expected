P-003

  × Unexpected "LParen", expected ")", ",", ":", ":=", "=>", "Identifier", "Operator", "_"
   ╭─[001-invalid-infix.pol:1:12]
 1 │ infix x + C(y) := D(x, y)
   ·            ─
   ╰────
